packages/emulation/libretro-pcsx-rearmed/package.mk

```diff
@@ -39,7 +39,7 @@ if [ "${ARCH}" = "arm" ]; then
   if [ "${DEVICE}" = "OdroidGoAdvance" ]; then
     sed -e "s|armv8-a|armv8-a+crc|" \
         -i ../Makefile.libretro
-    PKG_MAKE_OPTS_TARGET+=" platfrom=classic_armv8_a35"
+    PKG_MAKE_OPTS_TARGET+=" platform=classic_armv8_a35"
```

Comment:
Fixed a bug in this build script.

Reply:
Good find. This should be separated out. Does the fix result in a working addon for the target?